VK/VK/FriendsList/VC/FriendsListTableViewController.swift

```diff
@@ -28,6 +28,19 @@ final class FriendsListTableViewController: UITableViewController {
         loadFriendsFromDatabaseService()
     }
 
+    // MARK: - Public Methods
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard segue.identifier == Constants.friendsInfoSegueID,
+              let userPhotosViewController = segue.destination as? UserPhotosCollectionViewController
+        else { return }
+        guard let friendInfoIndexPath = tableView.indexPathForSelectedRow else { return }
+        let sortedKeys = sortedFriendsMap.keys.sorted()
+        let key = sortedKeys[friendInfoIndexPath.section]
+        guard let friendsInfo = sortedFriendsMap[key]?[friendInfoIndexPath.row] else { return }
```

Comment:
```suggestion
        guard let friendInfoIndexPath = tableView.indexPathForSelectedRow else { return }
        let sortedKeys = sortedFriendsMap.keys.sorted()
        let key = sortedKeys[friendInfoIndexPath.section]
        guard let friendsInfo = sortedFriendsMap[key]?[friendInfoIndexPath.row] else { return }
```
Может это в отдельный метод вынести?